org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/xml-declaration-snippets.json

```diff
@@ -19,6 +20,7 @@
       "<?xml version=\"${1|1.0,1.1|}\" encoding=\"${2|UTF-8,ISO-8859-1,Windows-1251,Windows-1252,Shift JIS,GB2312,EUC-KR|}\" standalone=\"${3|yes,no|}\"?>${0}"
     ],
     "label": "$description",
-    "description": "Insert XML Declaration with standalone"
+    "description": "Insert XML Declaration with standalone",
+	"sortText": "0"

```

Comment:
Very minor question, but is there a reason why both `sortText` are `0` instead of one being `0` and another being `1`?

Reply:
My idea was to group the snippet per type (here the type is XML declaration). Less we have values for sortText less it will be easier to manage sort.